cmd/nutmix/htlc_route_test.go

```diff
@@ -24,8 +24,17 @@ import (
 	"github.com/testcontainers/testcontainers-go/wait"
 )
 
-var correctPreimage = "0000000000000000000000000000000000000000000000000000000000000001"
-var incorrectPreimage = "0000000000000000000000000000000000000000000000000000000000000002"
+var correctPreimage string
+var incorrectPreimage string
+
+// there is a conditional check that the preimage is 32 bytes long
+func init() {
+	sum := sha256.Sum256([]byte("12345"))
+	correctPreimage = hex.EncodeToString(sum[:])
+
+	sum2 := sha256.Sum256([]byte("54321"))
+	incorrectPreimage = hex.EncodeToString(sum2[:])
+}
```

Comment:
There is an new check for the size of the preimages, the witness should have a 32 byte preimage.